riscof-plugin/sail_cSim/riscof_sail_cSim.py

```diff
@@ -0,0 +1,213 @@
+import os
```

Comment:
Run a linter like flake8 on this file, it's messy and not PEP-8 compliant

Reply:
213 lines of boilerplate that presumably will need to change over time as riscof evolves doesn't seem like a good design to me. The only thing Sail-specific in here is how to run the emulators; how to use a RISC-V compiler and objdump, how to create a Makefile and how to run this ISAC thing have nothing to do with Sail, that should be in your riscof library. This should only contain things that relate specifically to Sail compared with other platforms.